roles/test_collectd/tasks/main.yml

```diff
@@ -28,3 +28,11 @@
   register: stat
   changed_when: false
   failed_when: stat.stdout_lines|length == 0
+
+- name: TEST Check collectd memory limit
```

Comment:
Looks good! This is fine as-is, however, if it was to fail, I think it might be useful to have a tasks before this to check for whether the container was created with a memory limit. 

```
podman inspect not_collectd4 | jq '.[].Config.CreateCommand'
[
  "podman",
  "run",
  "-itd",
  "--memory",
  "512m",
  "--name",
  "not_collectd4",
  "alpine"
]

$ podman inspect not_collectd4 | jq '.[].HostConfig.Memory'
536870912

```

I'm not sure if we expect jq to be installed for this set of tests. If we don't expect it to be installed, you can disregard this feedback.

If it is already installed, then you can follow-up in another PR, if you want to extend the test in this way.